app/models/user.rb

```diff
@@ -48,7 +52,15 @@ def find_or_create_system_user(user_key)
     end
 
     def curators
-      RoleMapper.whois(CURATOR_GROUP)
+
+      #RoleMapper.whois(CURATOR_GROUP)
+
+      if curator_group = Role.where(name: User::CURATOR_GROUP).first
```

Comment:
I assume you can use here instead:

```ruby
Role.find_by(name: User::CURATOR_GROUP)
```

Reply:
Thanks David - Jim originally suggested something like that, but it ended up failing a bunch of tests. I think the rationale for where we ended up was accounting for the possibility that the curator group may not exist.

Reply:
If the curator group does not exist `find_by` should return `nil`.  `find_by!` (bang) raises an exception.

Reply:
Turns our I just misunderstood your suggestion - seems to work great, actually :)

Thanks again!